src/components/signup/constants.js

```diff
@@ -37,4 +47,18 @@ const userFormData = {
 	},
 };
 
-export { signUpData, userFormData };
+/**
+ * Defines the classes used in application.
+ *
+ * @constant
+ */
+const cssClasses = {
+	AUTH_LAYOUT_IMG: 'l-authLayout__img',
+	FORM_AUTH: 'a-formAuth',
+	FORM_AUTH_BTN: 'a-formAuth__button',
+	FORM_AUTH_CHECK_BTN: 'a-formAuth__check-btn',
+	SIGNUP_PAGE: 'p-signupPage',
+	SIGNUP_PAGE_HEADING: 'p-signupPage__heading',
+};
```

Comment:
do not save css styles to constants file, just the classes name directly in the component